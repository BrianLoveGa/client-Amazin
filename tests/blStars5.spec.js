import React from "react";
import HighStar from "../src/components/RatingSummary/highStar";
import ReactDom from 'react-dom';

// basic crash test 5 star

describe("This component test should render for 5 star without crashing", () => {
  it("Should render as expected!", () => {
    const div=document.createElement("div"); 
    ReactDom.render(<HighStar />, div);
  });
});