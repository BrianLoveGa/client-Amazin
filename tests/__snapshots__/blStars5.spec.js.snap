// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This snapshot test should pass for 4 stars Should render as expected! 1`] = `
<div
  className="highstar"
>
  <span
    className="gold"
    style={
      Object {
        "color": "rgb(255, 188, 40)",
      }
    }
  >
    ★
  </span>
  <span
    className="gold"
    style={
      Object {
        "color": "rgb(255, 188, 40)",
      }
    }
  >
    ★
  </span>
  <span
    className="gold"
    style={
      Object {
        "color": "rgb(255, 188, 40)",
      }
    }
  >
    ★
  </span>
  <span
    className="gold"
    style={
      Object {
        "color": "rgb(255, 188, 40)",
      }
    }
  >
    ★
  </span>
  <span
    className="gold"
    style={
      Object {
        "color": "rgb(255, 188, 40)",
      }
    }
  >
    ★
  </span>
  <span
    className="tiny"
    style={
      Object {
        "fontSize": "10px",
        "padding": "0px 20px 0px 5px",
        "position": "relative",
        "top": "-8px",
      }
    }
  >
     
    v
     
  </span>
  <a
    href="#"
    style={
      Object {
        "fontSize": "13px",
      }
    }
  >
     9,458 ratings
  </a>
  <span
    className="line"
    style={
      Object {
        "fontSize": "13px",
        "padding": "10px",
      }
    }
  >
     | 
  </span>
  <a
    href="#"
    style={
      Object {
        "fontSize": "13px",
      }
    }
  >
     167 answered questions
  </a>
</div>
`;
